scripts/search/settings.js

```diff
@@ -0,0 +1,84 @@
+// Algolia index settings.
+//
+// Changes to this file should be made with extreme care.
+module.exports = {
+
+    // Attributes for faceting are the properties to use for top-level filtering of results -- for
+    // example, whether a given record is returned for a "Docs" query, "Registry" query, and so on.
+    // https://www.algolia.com/doc/api-reference/api-parameters/attributesForFaceting/
+    getAttributesForFaceting() {
+        return [
+            "searchable(section)",
+        ];
+    },
+
+    // Synonyms allow us to surface content that may not directly match users' queries.
+    // https://www.algolia.com/doc/api-reference/api-methods/save-synonyms/#save-synonyms
+    getSynonyms() {
```

Comment:
can we pull on the synonyms from swifttype and add them here?

Reply:
Yep -- and I did, for the most part. It isn't quite an apples-to-apples thing, as these services are different, and many of the synonyms we added to Swiftype were account for typos, which Algolia [handles natively](https://www.algolia.com/doc/guides/managing-results/optimize-search-results/typo-tolerance/) (and I believe I verified all of them by hand to make sure), so there isn't any reason to bring those over directly. A few others aren't exactly "synonyms", either (e.g., "secret" and "passphrase", which refer to different things), so I left those out as well. But otherwise, they should look pretty similar, and I've added a few of my own as well.